package.json

```diff
@@ -37,7 +37,6 @@
   },
   "dependencies": {
     "chalk": "^2.4.1",
-    "eslint": "^4.2.0",
```

Comment:
Note that you can make it *both* a dep and a peer dep, and it’s still easy for users, but the peer dep will prevent conflicts with a higher level eslint.

Reply:
It will be installed if it’s not already in the tree and requireable; if it results in a duplicate, the one up the tree will fail the peer dep requirement.

I’m not sure how it works in yarn, but how it works in npm is the standard, so if it doesn’t work in yarn, yarn is broken :-)